src/components/Authentication/Login.js

```diff
@@ -122,7 +128,22 @@ function Login(props) {
               Sign In with Google
             </Button>
           </CardContent>
-          <Divider variant="middle" />
+          <CardContent>
+            <Button style={styles.facebookIconBtn}
```

Comment:
Instead of adding styles can we use the color prop?

Reply:
the styling for facebookIconBtn also includes margins tho

Reply:
Can we remove background color (on google and facebook) and use the color prop instead?

Reply:
the color prop on buttons only supports the default color
https://material-ui.com/zh/api/button/

Reply:
updated, use the theme provider for the background color